beast/tools/read_beast_data.py

```diff
@@ -124,6 +124,7 @@ def read_sed_data(
        the set of parameters to extract
        default = [Av, Rv, f_A, M_ini, logA, Z, distance]
        If set to None, return the list of possible parameters
+       If set to 'all', read in all parameters
```

Comment:
Need a bit more documentation on how `None` is different from `all`.

Reply:
Yeah, that was a bit confusing.  I made a separate keyword for returning the list of parameters, so hopefully that's more clear.